### PR TITLE
Update Rust Nightly Version

### PR DESCRIPTION
**Pull request by ChristianBeilschmidt**
- Fix a new Rust Nightly version

Auto-generated PR

### CODE REVIEW

**Comment by ChristianBeilschmidt** [comment]
[Update Rust Nightly](https://trello.com/c/JDjW391n/286-update-rust-nightly)

File: datatypes/src/collections/batch_builder.rs [hunk 1]
```diff
@@ -144,6 +144,7 @@ impl RawFeatureCollectionBuilder {
         self.types.clone()
     }
 
+    #[allow(clippy::unnecessary_wraps)]
```
Comment (michaelmattig) [inline datatypes/src/collections/batch_builder.rs:147]: why is this needed? It says
> Checks for private functions that only return Ok or Some.

 but this is not a private function

**Review by michaelmattig — COMMENTED** [review]
.

File: datatypes/src/collections/batch_builder.rs (hunk 1 above)
Comment (ChristianBeilschmidt) [reply datatypes/src/collections/batch_builder.rs:147]: The description maybe not be accurate but Clippy warns if I remove it.
